Lab Member YAMLs/Templates/Full Name with Spaces.yaml

```diff
@@ -0,0 +1,46 @@
+name:
+birthday: MM.DD
+position: (Job Title) (Graduate Student) (Postoctoral Research Scholar/Fellow)
+affiliations: >
+  Weill Cornell Physiology, Biophysics, and Systems Biology PhD Program
```

Comment:
Could we make this a list instead of text? So:
```yaml
affiliations:
  - Weill Cornell Biochemistry and Structural Biology PhD Program
  - Tri-Institutional PhD Program in Computational Biology and Medicine
  - Tri-Institutional PhD Program in Chemical Biology
  - Louis V. Gerstner, Jr. Graduate School of Biomedical Sciences
  - Berlin Institute of Health (BIH) Einstein Foundation - Stiftung Charité Group
  - Joint Mentorship with Name (URL to lab)
```